src/pages/settings/Payments/PaymentsPage.js

```diff
@@ -145,6 +226,97 @@ class PaymentsPage extends React.Component {
                         }}
                         onItemSelected={method => this.addPaymentMethodTypePressed(method)}
                     />
+                    <Popover
+                        isVisible={this.state.shouldShowDefaultDeleteMenu}
+                        onClose={this.hideDefaultDeleteMenu}
+                        anchorPosition={{
+                            top: this.state.anchorPositionTop,
+                            left: this.state.anchorPositionLeft,
+                        }}
+                    >
+                        <View
+                            style={[
+                                styles.m5,
+                                !this.props.isSmallScreenWidth ? styles.defaultDeletePopover : '',
+                            ]}
+                        >
+                            {this.props.isSmallScreenWidth && (
+                                <MenuItem
+                                    title={this.state.formattedSelectedPaymentMethod.title}
+                                    icon={Expensicons.Bank}
+                                    description={this.state.formattedSelectedPaymentMethod.description}
+                                    onPress={() => {}}
```

Comment:
Are we kind of stretching the definition of a `MenuItem` here by using this as a "title" and not passing an `onPress`? Maybe we should remove the requirement of that callback and make sure the item should not behave like a button.

Reply:
Yeah I agree with that. MenuItem is perfectly formatted for this case so I think making the onPress optional is a good idea

Reply:
Oh JK at some point since I made this PR somebody made it not required anymore. Removed the prop.